Core/Services/CarService.cs

```diff
@@ -73,14 +73,21 @@ private async Task<bool> IsCarExist(Car newCar)
                     newCar.TechnicalPassport));
         }
 
-        public async Task<bool> CheckIsCarBelongsToUserByIds(int carId, string userId)
+        public async Task<bool> CheckIsUserVerifiedByIdsAsync(int carId, string userId)
```

Comment:
probably we should use "CheckIsUserCarVerifiedByIdsAsync"?

Reply:
I try to avoid the word "Car" because it is in the name of the service